src/components/navbar.tsx

```diff
@@ -1,63 +1,82 @@
+import { useRouter } from "next/router";
+import Navbar from "react-bootstrap/Navbar";
+import Nav from "react-bootstrap/Nav";
+import Image from "next/image";
+import logo from "../public/Logo.svg";
 import Link from "next/link";
-import { FunctionComponent } from "react";
 
-interface NavbarProps {
-  setShowSettings: (show: boolean) => void;
-}
+const VerticalNavbar = () => {
+  const router = useRouter();
+
+  const navItems = [
+    { path: "/room/1", label: "Raum 1" },
+    { path: "/room/2", label: "Raum 2" },
+  ];
+
+  const isActive = (path: string) => {
+    return router.asPath === path;
+  };
 
-const Navbar: FunctionComponent<NavbarProps> = ({ setShowSettings }) => {
   return (
-    <nav className="navbar navbar-expand-lg navbar-dark bg-dark">
-      <div className="container">
-        <Link href="/">
-          <span className="navbar-brand">TUC Forum</span>
-        </Link>
-        <button
-          className="navbar-toggler"
-          type="button"
-          data-bs-toggle="collapse"
-          data-bs-target="#navbarNav"
-          aria-controls="navbarNav"
-          aria-expanded="false"
-          aria-label="Toggle navigation"
+    <Navbar bg="primary" className="d-flex flex-column vh-100">
+      <Navbar.Brand
+        as={Link}
+        href="/"
+        className="mx-auto"
+        style={{ cursor: "pointer" }}
+      >
+        <Image
+          src={logo}
+          alt="Logo"
+          width={50}
+          height={50}
+          className="rounded"
+        />
+      </Navbar.Brand>
+      <Nav className="flex-column d-flex flex-grow-1">
```

Comment:
```suggestion
      <Nav className="flex-column d-flex flex-grow-1 align-items-center">
```
(und dann noch die `mx-auto` Einträge entfernen bei den children)